src/Home.js

```diff
@@ -118,6 +119,13 @@ class Home extends Component {
       );
     const isCalculateDisabled = !this.canBeSubmitted();
 
+    let displaySummary =
+      calculationsTable.length > 1 ? (
```

Comment:
We have this check in two places - wonder whether it would be better to have one of...
- A check in the return to conditionally render the components
- A single wrapper component for the summary + details